NDB.Covid19/NDB.Covid19/Locales/en.json

```diff
@@ -1,7 +1,5 @@
 {
-  "LAUNCHER_PAGE_START_BTN_NB": "Start på bokmål",
-  "LAUNCHER_PAGE_START_BTN_NN": "Start på nynorsk",
-  "LAUNCHER_PAGE_CONTINUE_IN_ENG": "Continue in English",
+  "LAUNCHER_PAGE_START_BTN": "Start",
```

Comment:
Have you updated the excel sheet AND the wiki page accordingly?

Reply:
Yes